R/murphy_diagram.R

```diff
@@ -0,0 +1,80 @@
+#' Murphy diagram
+#'
+#' Murphy diagram for elementary scores. Can be used to compare performance of one or multiple models across ranges of thetas for elementary scores. If the plot needs to be customized, set \code{plot = FALSE} to get the resulting data instead of the plot.
+#'
+#' @importFrom graphics matplot legend
+#' @param actual Observed values.
+#' @param predicted Predicted values.
+#' @param w Optional case weights.
+#' @param alpha Level of expectile resp. quantile.
+#' @param theta Vector of theta values.
+#' @param type One of "expectile", "quantile".
```

Comment:
The default is a vector, so vetors are allowed, too?

Reply:
`type` has "expectile" as single default, even if it seems to be a vector. This is an R thing to show all possible input values. Other values raise an error in `match.arg()`. Is "expectiles" better than "expectile"?

Reply:
:bulb:
"expectile**s**" was a misspelling of mine.